src/storage.ts

```diff
@@ -388,9 +389,15 @@ export class Storage extends Service {
     options = Object.assign({}, options, {
       apiEndpoint: options.apiEndpoint || 'storage.googleapis.com',
     });
-    const url =
-      process.env.STORAGE_EMULATOR_HOST ||
-      `https://${options.apiEndpoint}/storage/v1`;
+
+    // Overrides options.apiEndpoint.
```

Comment:
If the user provides "options.apiEndpoint", that should override the env var. Similar to options.keyFilename beating ADC.

Reply:
@jkwlui